x-pack/plugins/watcher/common/constants/watch_states.ts

```diff
@@ -5,26 +5,16 @@
  * 2.0.
  */
 
-import { i18n } from '@kbn/i18n';
-
+// These values are returned by the API and depended upon in both server-side
+// and client-side logic. TS will use these values to determine correctness.
```

Comment:
What do you think about using snakecase for the values  (`ok`, `config_error`, `disabled`) since these will not surface in the UI?